frontend/src/reducers/productReducers.js

```diff
@@ -0,0 +1,21 @@
+const {
+  PRODUCT_LIST_REQUEST,
+  PRODUCT_LIST_SUCCESS,
+  PRODUCT_LIST_FAIL,
+} = require('../constants/productConstants');
```

Comment:
According to ES6 it should be:

**import** {
	PRODUCT_LIST_REQUEST,
	PRODUCT_LIST_SUCCESS,
	PRODUCT_LIST_FAIL,
} **from** "../constants/productConstants";